apps/web/app/[locale]/dashboard/team-dashboard/[teamId]/components/team-stats-table.tsx

```
@@ -3,85 +3,185 @@
 import { ChevronLeft, ChevronRight, ChevronsLeft, ChevronsRight } from 'lucide-react';
-import { members } from '../data/mock-data';
+import { useState } from 'react';
 
-const getProgressColor = (activityLevel: string) => {
-	const level = parseInt(activityLevel, 10);
-	if (isNaN(level) || level < 0) return 'bg-gray-300';
-	if (level > 100) return 'bg-green-500';
-	if (level <= 20) return 'bg-red-500';
-	if (level <= 50) return 'bg-yellow-500';
+const getProgressColor = (activityLevel: number) => {
+	if (isNaN(activityLevel) || activityLevel < 0) return 'bg-gray-300';
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)